paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -152,27 +130,19 @@ public class PaparazziPlugin : Plugin<Project> {
         val gradleHomeDir = projectDirectory.dir(project.gradle.gradleUserHomeDir.path)
 
         task.packageName.set(android.packageName())
-        task.artifactFiles.from(packageAwareArtifactFiles)
+        task.artifactFiles.from(sources.packageAwareArtifactFiles)
         task.nonTransitiveRClassEnabled.set(nonTransitiveRClassEnabled)
         task.targetSdkVersion.set(android.targetSdkVersion())
-
-        val localResourcePaths = localResourceDirs
-          .map { layers -> layers.flatten() }
-          .map { dirs -> dirs.map { projectDirectory.relativize(it.asFile) }.asReversed() }
-
-        task.projectResourceDirs.set(localResourcePaths)
-        task.moduleResourceDirs.set(moduleResourceDirs.relativize(projectDirectory))
-        task.aarExplodedDirs.set(aarExplodedDirs.relativize(gradleHomeDir))
+        task.projectResourceDirs.set(
+          sources.localResourceDirs.relativize(projectDirectory).reversed()
```

Comment:
Have you considered these in, OR an extension of `AndroidVarianSources`?

Reply:
Ah, yea the reversed stuff is for resource and asset priority ordering, so makes sense to move it there.  Done.